api/v1alpha1/profilesubscription_types.go

```diff
@@ -49,6 +51,26 @@ type ProfileSubscriptionSpec struct {
 	// ValuesFrom holds references to resources containing values for the Helm chart specified in the first artifact
 	// +optional
 	ValuesFrom []helmv2.ValuesReference `json:"valuesFrom,omitempty"`
+
+	// ProfileCatalogDescription defines properties of the catalog this profile is from
+	ProfileCatalogDescription *ProfileCatalogDescription `json:"profile_catalog_description,omitempty"`
+}
+
+// ProfileCatalogDescription defines properties of the catalog this profile is from
+type ProfileCatalogDescription struct {
+	// Version defines the version of the catalog to get the profile from
+	Version string `json:"version,omitempty"`
+
+	// Catalog defines the name of the catalog to get the profile from
+	Catalog string `json:"catalog,omitempty"`
+
+	// ProfileName defines the name of the profile
+	ProfileName string `json:"profile_name,omitempty"`
```

Comment:
Since we don't say `CatalogName`, could we rename `ProfileName` to `Profile`? What do you think?

Reply:
Sure, that makes sense.